jovian/utils/api.py

```diff
@@ -154,3 +154,30 @@ def commit_records(gist_slug, tracking_slugs, version=None):
         return res.json()['data']
     else:
         raise ApiError('Data logging failed: ' + _pretty(res))
+
+
+def notify_on_slack(data):
```

Comment:
Better call it `post_slack_message`, to avoid confusion between this and the `notify` method.